Framework/Algorithms/src/CombineDiffCal.cpp

```diff
@@ -70,9 +70,60 @@ API::ITableWorkspace_sptr CombineDiffCal::sortTableWorkspace(DataObjects::TableW
   return alg->getProperty("OutputWorkspace");
 }
 
+bool findColumn(const std::vector<std::string> columnNames, const std::string name) {
+  return std::find(columnNames.begin(), columnNames.end(), name) != columnNames.end();
+}
+
+int hasAllExpectedDifcColumns(const DataObjects::TableWorkspace_sptr ws) {
```

Comment:
I suppose I could, but isn't catching errors expensive?  I suppose it doesn't matter since that means its in a fail state anyway but this should still be fine, right?